graph-builder/tests/net/quay_io/mod.rs

```diff
@@ -324,3 +328,36 @@ fn fetch_release_public_must_succeed_with_schemes_missing_http_https() {
     .map(|url| registry::Registry::try_from_str(url).unwrap())
     .for_each(test);
 }
+
+#[test]
+fn fetch_relesae_with_cyclic_metadata_fails() -> Fallible<()> {
```

Comment:
typo: s/relesae/release/

Reply:
_relesae_ would be a nice plural form though ;-